src/components/App/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Ah I think, my comment on the animations may have been answered. I am guessing you were planning on having different routes to render different components and then one would swivel or pop in front of the other?